.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "external/qdldl"]
+	path = external/qdldl
```

Comment:
Maybe it shouldn't be a submodule to avoid being build scripts in other repositories?

Reply:
Thats a good question. I was following the example of OSQP, but I didn't think about that affecting dependencies.